src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/DefaultTagHelperContent.cs

```diff
@@ -342,7 +344,7 @@ public override Encoding Encoding
             public bool IsEmptyOrWhiteSpace { get; private set; } = true;
 
 #if NETSTANDARD1_5
-            // This is an abstract method in DNXCore
+            // This is an abstract method in .NET Core but NullHtmlEncoder doesn't need it.
             public override void Write(char value)
```

Comment:
Can't this still get called if a users `IHtmlContent` calls `writer.Write((char)...)` in its `WriteTo(writer, encoder)` method?